controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/ApplicationList.java

```diff
@@ -122,8 +122,8 @@ public ApplicationList hasDeployment() {
     }
 
     /** Returns the subset of applications which started failing after the given instant */
```

Comment:
Update Javadoc too.

Reply:
Argh, yes, that was what I intended to do :p

Reply:
Ah, but I did!